twisted/internet/iocpreactor/tcp.py

```diff
@@ -59,7 +60,13 @@ def getHandle(self):
 
     def dataReceived(self, rbuffer):
         # XXX: some day, we'll have protocols that can handle raw buffers
-        self.protocol.dataReceived(str(rbuffer))
+        if not isinstance(rbuffer, bytes):
+            if _PY3:
```

Comment:
There was a Windows Python 3 buildbot, but adiroiban disabled it and deleted it.
Right now the tox.ini file records coverage, and the appveyor job which runs tox uploads the coverage report to codecov.io.  However, the appveyor job is only doing Windows Python 2.

We can't enable a Windows Python 3 buildbot / appveyor just yet, because some of the jobs hang, but this patch is one big step in getting us closer.